docs/cli-existing-project.md

```diff
@@ -44,22 +43,7 @@ In this guide, you'll propose the deployment on the command line and then approv
   * [Node Version >=16.16.0](https://nodejs.org/en/download). (Run `node -v` to see your current version).
   * [Rust Compiler](https://www.rust-lang.org/tools/install)
 
-## 2. Install Sphinx Foundry Fork
-For now, Sphinx requires a [fork of Foundry](https://github.com/sphinx-labs/foundry) that resolves several bugs that conflict with Sphinx. We're working on getting these fixed in the official Foundry repo.
-
-If you are using macOS, you will first need to install the Xcode Command Line Tools:
-```
-xcode-select --install
-```
-
-Then, install the Sphinx Foundry fork with the following command. The installation process may take several minutes to complete.
-```
-foundryup --repo sphinx-labs/foundry --branch sphinx-patch-v0.1.0
-```
-
-> If you run into problems installing our Foundry fork, [try steps in the troubleshooting guide](https://github.com/sphinx-labs/sphinx/blob/main/docs/troubleshooting-guide.md#installing-sphinxs-foundry-fork). If you are still unable to resolve the issue, please reach out in the [Discord](https://discord.gg/7Gc3DK33Np). We're always happy to help.
```

Comment:
> If you run into problems installing our Foundry fork, [try steps in the troubleshooting guide](https://github.com/sphinx-labs/sphinx/blob/main/docs/troubleshooting-guide.md#installing-sphinxs-foundry-fork). If you are still unable to resolve the issue, please reach out in the [Discord](https://discord.gg/7Gc3DK33Np). We're always happy to help.

Should we keep this? Looks like it's removed in this PR

Reply:
If you don't think the troubleshooting guide is necessary anymore, can you delete it?

Reply:
I don’t think the troubleshooting guide is necessary. I’ll remove it. We can reintroduce it if we see people running into problems with the installation process.